style/MainView.mcss

```diff
@@ -131,6 +131,33 @@
   top: -1px;
 }
 
+.logo-container-outer {
```

Comment:
I wonder could go wrong since I didn't use `:global(.logo-container-outer)` here?  It seems to be working, but...